src/Aether/Devices/Drivers/Sgp4x.cs

```diff
@@ -0,0 +1,174 @@
+﻿using System.Device.I2c;
+using UnitsNet;
+
+namespace Aether.Devices.Drivers
+{
+    /// <summary>
+    /// Air Quality and Gas VOC Sensor SGP4x
+    /// </summary>
+    public sealed class Sgp4x : System.IDisposable
+    {
+        /// <summary>
+        /// The default I²C address of this device.
+        /// </summary>
+        public const int DefaultI2cAddress = 0x59;
+
+        private readonly I2cDevice _device;
+
+        /// <summary>
+        /// Instantiates a new <see cref="Sgp4x"/>.
+        /// </summary>
+        /// <param name="device">The I²C device to operate on.</param>
+        public Sgp4x(I2cDevice device)
+        {
+            _device = device;
+            Reset();
+        }
+
+        /// <inheritdoc/>
+        public void Dispose() =>
+            _device.Dispose();
+
+        /// <summary>
+        /// Resets the device.
+        /// </summary>
+        public void Reset()
+        {
+            _device.WriteByte(0x00);
+            _device.WriteByte(0x06);
+            Thread.Sleep(1);
+        }
+
+        /// <summary>
+        /// Gets the serial number of the device.
+        /// </summary>
+        /// <returns>The serial number of the device.</returns>
+        public ReadOnlySpan<byte> GetSerialNumber()
+        {
+            Span<byte> getSerialNumberCommand = stackalloc byte[2] { 0x36, 0x82 };
```

Comment:
It doesn't matter for getting a serial number, but optimization tip:

```suggestion
            ReadOnlySpan<byte> getSerialNumberCommand = new byte[2] { 0x36, 0x82 };
```

Both the `ReadOnlySpan` and the `new byte[]` bits are important.

See the difference on [SharpLab](https://sharplab.io/#v2:EYLgtghglgdgNAFxBAzmAPgAQEwEYCwAUJgAwAEmuAdAEoCuMCUYAplQMID2YADlADYsATgGVhANygBjFigDcRIpgDMFXADYK2MuzIBvImSMVVlTZgAsZAHIAKAJT7DxlzRYQAJgHkY/AJ4iPBAwADzAfggsAHxkwGQAvGQoCBBSANYQ/PycUrERLADaALr6ZLhwZNgVqgC+CoQuLgAinCLcLAgAFrAA5rbA9vUuNc5Go2QA9BNkXVAoZD1CEMDzXSxk4pl06wBmQtxkQu4eALScvn5kHhApVOMqauZWALIOTg2NRm6ePv6BwWF8jE4okYCwAO55SLFUrlSrVMh1cbNVrtWYwPoDIbGEYfMZ4ybTABWdGSZAgVzoYDAlwQnDIPCO4hYjDInB4TDAUAAXr02XQEDNOusWUwjkK+R5tjN6TB6SgoB4WCcWDsdiwpAgUHcCQVnh1OpwPABJXj8Wz6rpG008fheDlQc7a6ycY2+WC9exFe6mDQUKwtNqsdF9b7eC7/UIAISBSSEUns4wMBNxNSAA).

This can be applied in a few places here.